src/pages/workspace/accounting/netsuite/import/NetSuiteImportCustomFieldNew/NetSuiteImportAddCustomListPage.tsx

```diff
@@ -36,7 +36,9 @@ function NetSuiteImportAddCustomListPage({policy}: WithPolicyConnectionsProps) {
     const customLists = useMemo(() => config?.syncOptions?.customLists ?? [], [config?.syncOptions]);
 
     const handleFinishStep = useCallback(() => {
-        Navigation.goBack(ROUTES.POLICY_ACCOUNTING_NETSUITE_IMPORT_CUSTOM_FIELD_MAPPING.getRoute(policyID, CONST.NETSUITE_CONFIG.IMPORT_CUSTOM_FIELDS.CUSTOM_LISTS));
+        InteractionManager.runAfterInteractions(() => {
```

Comment:
Could you explain your investigation by using `InteractionManager.runAfterInteractions`? As far as I am concerned we don't have animation run beside the navigation and we don't have any issue with that.

Reply:
It was proposed in https://github.com/Expensify/App/issues/43410#issuecomment-2162088604 (PR: https://github.com/Expensify/App/pull/45691) and https://github.com/Expensify/App/issues/37534#issuecomment-1971812156 as a way to wait until optimistic data is applied. I've also tried `SequentialQueue.waitForIdle()`, but it waits way longer in offline mode.

Reply:
Thanks @CyberAndrii! I've noticed this seems to happen in other places too, this might be related to race conditions.